sdk/core/azure-core/inc/azure/core/core.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we move this simplified header up to `https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/core/azure-core` OR
`https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/core/azure-core/inc/azure` rather than nested in the `/core/` directory next to all the other "specific" headers?

Reply:
What would the customer header include look like?
It would be good if it was simple, like:
```C++
#include <azure/core.hpp>
```

Reply:
That makes sense.  A general guideline should be added to describe what each package needs to do.  

Each package should have a general include that resides at their package level.  That general header would be at the level of the package sub-service
```C++
//Include azure-**core**
#include <azure/core.hpp>

//Include just azure-storage-**blobs**
#include <azure/storage/blobs.hpp>

//Include just azure-storage-files-**datalake**
#include <azure/storage/files/datalake.hpp>

//Include just azure-storage-files-**shares**
#include <azure/storage/files/shares.hpp>

//Include just azure-keyvault-**keys**
#include <azure/keyvault/keys.hpp>

```

Reply:
Why do you have `files` folder for storage datalake and shares, and you don't have `blobs` folder for blobs?

Let's take it to Design meeting.
The current simplified headers from storage are:

```cpp
#include "azure/storage/blobs/blob.hpp"
#include "azure/storage/files/datalake/datalake.hpp"
#include "azure/storage/files/shares/shares.hpp"
```

Reply:
cc @katmsft, @Jinming-Hu